app/packages/state/src/recoil/selectors.ts

```diff
@@ -147,6 +147,19 @@ export const datasetAppConfig = graphQLSyncFragmentAtom<
   }
 );
 
+export const getDisableFrameFiltering = selector<boolean>({
+  key: "disableFrameFiltering",
+  get: ({ get }) => {
+    const datasetDisableFrameFiltering = Boolean(
+      get(datasetAppConfig)?.disableFrameFiltering
+    );
+    const globalDisableFrameFiltering = Boolean(
+      get(appConfigOption({ modal: true, key: "disableFrameFiltering" }))
+    );
+    return datasetDisableFrameFiltering || globalDisableFrameFiltering;
```

Comment:
TODO: update this to instead check if the dataset is non null (use it if it isn't) otherwise use the global value which will default to false or be whatever the client set.